src/mm-datepicker/mm-datepicker.js

```diff
@@ -58,6 +58,9 @@
 			state:'closed',
 			valign: { value: "center", reflect: true },
 			align: { value: "center", reflect: true },
+
+			closeText: "Close",
```

Comment:
super nitpicky, but could you change these to be closeLabel, saveLabel?

other than that +1